packages/stable/src/api/documents/types.ts

```diff
@@ -0,0 +1,369 @@
+import { CogniteExternalId, CogniteInternalId } from '@cognite/sdk-core';
+
+export type DocumentSearchRequest = DocumentFilterOption &
+  DocumentSearch &
+  DocumentAggregates &
+  DocumentSort &
+  DocumentSearchLimit;
+
+/**
+ * Filter with exact match
+ */
+export interface DocumentFilterOption {
+  /** Filter with exact match */
+  filter?: DocumentFilter;
+}
+
+export interface DocumentSearch {
+  search?: { query?: string; highlight?: boolean };
+}
+
+export interface DocumentAggregates {
+  aggregates?: DocumentCountAggregate[];
+}
+
+export interface DocumentSort {
+  /**
+   * List of fields to sort by, currently only supports 1 field.
+   * Syntax: `["<fieldname>:asc|desc"]`. Default sort order is `asc` with short syntax `["<fieldname>"]`.
+   *
+   * @example ["externalId:desc"]
+   */
+  sort?: string[];
```

Comment:
I also notice this is the sort structure for the playground API, not the v1 API. I think we need to redo this file to get things correct and in sync.

Reply:
we could also write one monster request that touches every field. If the api returns 400, then we know its a little off.

Reply:
I have fixed the sort structured now, and added an integration test for it. There are also integration tests for the other major things.